src/systems/ec/implementations/vanilla/packages/ec-component-form-checkbox/ec-component-form-checkbox.scss

```diff
@@ -0,0 +1,102 @@
+/**
+ * Checkbox component
+ * @define checkbox
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
```

Comment:
since `.ecl-checkbox__icon` is applied to the `<Icon>` component, you must make sure that the SCSS code for the icon has already been included with `@include check-imports(('ec-component-icon'));` (as https://github.com/ec-europa/europa-component-library/blob/v2/src/systems/ec/implementations/vanilla/packages/ec-component-button/ec-component-button.scss#L9)